engines/dfc_provider/app/serializers/dfc_provider/catalog_item_serializer.rb

```diff
@@ -14,10 +14,9 @@ class CatalogItemSerializer < BaseSerializer
              key: 'dfc:offeredThrough'
 
     def id
-      dfc_provider_routes.api_dfc_provider_enterprise_catalog_item_url(
+      api_v0_dfc_provider_enterprise_catalog_item_path(
```

Comment:
Actually we should use paths here. It is not clean yet for JSON-LD full support, but our next job will be to work on that.